common/src/main/java/org/opensearch/ml/common/parameter/AnomalyLocalizationOutput.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.ml.engine.algorithms.anomalylocalization;
+package org.opensearch.ml.common.parameter;
```

Comment:
I have been wondering this ml.common.parameter directory is bloating quickly and it already includes so many files. Should we considering using subdirectories like model, task, <algorithmA>, <algorithmB>, etc?

Reply:
Yes, that's the plan. The refactoring may impact quite a lot places. Let's finish the testing first, then move these classes to different packages.